include/trackcpp/accelerator.h

```diff
@@ -27,10 +27,10 @@ class Accelerator {
   // energy < electron_rest_energy -> energy = electron_rest_energy:
   Accelerator(const double& energy=-1);
   double                  energy;              // [eV]
-  bool                    cavity_on;
```

Comment:
these variables were not being initialized. pyaccel was taking care of it but this PR fixes this issue.
also deleted unnecessary semicolons that were generating compilaion warnings.